.github/workflows/ci.yml

```diff
@@ -23,3 +23,25 @@ jobs:
 
     - name: Run PHP lint
       run: composer ci:php:lint
+
+  php-code-sniffer:
+    name: PHP Code Sniffer
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version:
+          - 7.2
+          - 7.3
+          - 7.4
```

Comment:
For sniffing, one PHP version should suffice, as the result will not depend on the PHP version running the sniffer.